src/cljs/harja/ui/grid/perus.cljs

```diff
@@ -1005,27 +1005,40 @@
        ")"))
 
 (defn arvo-ja-nappi
-  "Jos annettu ehto on true, näyttää gridissä napin, jolla kenttää voidaan muokata.
-   Jos ehto on false, piirtää kentän arvon tekstinä sekä napin kynä-ikonilla,
-   jolla arvoa voi muokata."
-  [{:keys [ehto-fn nappi-teksti nappi-optiot
-           uusi-fn muokkaa-fn arvo voi-muokata?
-           muokkaa-ikoni ikoninappi?] :as optiot}]
-  (if voi-muokata?
+  "Piirtää arvon ja napin, tai pelkän napin optioista riippuen.
```

Comment:
Rajapinta muuttui hieman, lähinnä nimeämiset. Komponentti oletti aiemmin, että siinä näkyvällä napilla muokataan arvoa. Enää ei oleteta näin, vaan kyseessä on geneerinen arvo ja nappi.

Lisäksi voi nyt helposti määrittää, piirretäänkö arvo ja nappi, pelkkä nappi, vai pelkkä arvo.